src/CoreBundle/EventListener/SubSystemBootListener.php

```diff
@@ -111,22 +111,28 @@ public function boot(): void
             return;
         }
 
-        if (!$this->connection->getSchemaManager()->tablesExist(
-            [
-                'tl_metamodel',
-                'tl_metamodel_dca',
-                'tl_metamodel_dca_sortgroup',
-                'tl_metamodel_dcasetting',
-                'tl_metamodel_dcasetting_condition',
-                'tl_metamodel_attribute',
-                'tl_metamodel_filter',
-                'tl_metamodel_filtersetting',
-                'tl_metamodel_rendersettings',
-                'tl_metamodel_rendersetting',
-                'tl_metamodel_dca_combine',
-            ]
-        )) {
-            $this->logger->error('MetaModels startup interrupted. Not all MetaModels tables have been created.');
+        try {
+            if (!$this->connection->getSchemaManager()->tablesExist(
+                [
+                    'tl_metamodel',
+                    'tl_metamodel_dca',
+                    'tl_metamodel_dca_sortgroup',
+                    'tl_metamodel_dcasetting',
+                    'tl_metamodel_dcasetting_condition',
+                    'tl_metamodel_attribute',
+                    'tl_metamodel_filter',
+                    'tl_metamodel_filtersetting',
+                    'tl_metamodel_rendersettings',
+                    'tl_metamodel_rendersetting',
+                    'tl_metamodel_dca_combine',
+                ]
+            )) {
+                $this->logger->error('MetaModels startup interrupted. Not all MetaModels tables have been created.');
+                return;
+            }
+        } catch (\Throwable $throwable) {
```

Comment:
Any reason silencing `\Throwable` and not only `Doctrine\DBAL\DBALException`?

Reply:
No serious reason, aside from the fact that we want to interrupt booting entirely when anything here goes wrong.
We already did so in Contao 3, as there were multiple reasons where we would want to abstain.

I do not know what the upcoming changes in doctrine will bring, as they are removing the single common repo and therefore thought it could be wise to simply abstain on all errornous situations.